.github/workflows/ci.yaml

```diff
@@ -14,6 +14,9 @@ jobs:
         ruby-version:
         - '2.7'
         - '3.1'
```

Comment:
Side question, but is there a reason we don't have ruby 3.0 here?

Reply:
I was thikning 3.0 because that's what we ship with.

Reply:
lets do it!

Reply:
I'll make a separate PR